README.md

```diff
@@ -72,12 +62,11 @@ cloud-build-local --config=cloudbuild.yaml --dryrun=false .
 gcloud auth application-default login
 
 go run main.go \
-    --bq_dataset=google.com:datcom-store-dev.dc_kg_2020_04_13_02_32_53 \
-    --bt_table=borgcron_2020_04_13_02_32_53 \
+    --bq_dataset=$(head -1 deployment/bigquery.txt) \
+    --bt_table=$(head -1 deployment/bigtable.txt) \
     --bt_project=google.com:datcom-store-dev \
```

Comment:
Putting this in a single line is more easily copyable in my experience. Like:

```
go run ...
```

Reply:
ditto for above

Reply:
I've actually had the reverse experience, since a very long line requires lots of horizontal scrolling to get to the end. It was probably harder when you had to update the arguments? Either way, I'm a fan of scripts :)